website/docs/api/architectures.mdx

```diff
@@ -481,6 +481,318 @@ The other arguments are shared between all versions.
 
 </Accordion>
 
+## Curated transformer architectures {id="curated-trf",source="https://github.com/explosion/spacy-curated-transformers/blob/main/spacy_curated_transformers/models/architectures.py"}
+
+The following architectures are provided by the package
+[`spacy-curated-transformers`](https://github.com/explosion/spacy-curated-transformers). See the
+[usage documentation](/usage/embeddings-transformers#transformers) for how to
+integrate the architectures into your training config.
+
+<Infobox variant="warning">
+
+Note that in order to use these architectures in your config, you need to
+install the
+[`spacy-transformers`](https://github.com/explosion/spacy-transformers). See the
+[installation docs](/usage/embeddings-transformers#transformers-installation)
+for details and system requirements.
+
+</Infobox>
+
+### spacy-curated-transformers.AlbertTransformer.v1
+
+Construct an ALBERT transformer model.
+
+| Name                           | Description                                                                 |
+|--------------------------------|-----------------------------------------------------------------------------|
+| `vocab_size`                   | Vocabulary size. ~~int~~                                                    |
+| `with_spans`                   | Callback that constructs a span generator model. ~~Callable~~               |
+| `with_spans`                   | piece_encoder (Model) ~~Callable~~                                          |
+| `with_spans`                   | The piece encoder to segment input tokens. ~~Callable~~                     |
```

Comment:
It looks like something went wrong with the generation/alignment for the names and descriptions?

Reply:
@vin-ivar : can you have another look at these?

Reply:
Fixed.

Reply:
There are still a ton of problems with the tables, e.g.:
https://github.com/explosion/spaCy/pull/12677/files#diff-2514925cc24ac1e94f41e2a3d3bd3b8936563667ba04873cc843e753a32f5cfdR513-R514